.github/actions/upgrade-indexer/upgrade-indexer.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Gets the absolute path of the script, used to load the common.sh file
+function get_absolute_path() {
+    RELATIVE_PATH="$(dirname -- "${BASH_SOURCE[0]}")"
+    ABSOLUTE_PATH="$(cd -- "$RELATIVE_PATH" && pwd)"
```

Comment:
Use ${} for the variables, please check the rest of the script.

Reply:
Done. This code snippet has been changed.